src/static/js/modules/loan-comparison/components/loan-output-table-cell.js

```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+var common = require('../common');
+var LoanOutput = require('./loan-output');
+
+var LoanOutputCell = React.createClass({
+    displayClassNames: function(loan, prop, type) {
+        var loanID = loan['id'] === 0 ? 'a' : 'b',
+            propResult = prop + '-display-' + loanID,
+            loanScenario = 'lc-result-' + loanID,
+            classes = propResult + ' ' + loanScenario + ' lc-result';
+        if (type === 'primary') {
```

Comment:
Do we still need the loan-specific class names? If not, we could move these classes (if they're still being used for styling) & the `primary` class to the rows instead